crates/sui-indexer/src/metrics.rs

```diff
@@ -97,6 +98,13 @@ impl IndexerCheckpointHandlerMetrics {
                 registry,
             )
             .unwrap(),
+            transaction_per_checkpoint: register_histogram_with_registry!(
+                "transaction_per_checkpoint",
+                "Number of transactions per checkpoint",
+                vec![1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0, 2000.0, 5000.0],
```

Comment:
I know we have a couple histogram vectors. Should we standardize these? or at least not hardcode it?

Reply:
the latency one and the counter here is not different, this vector is the only used one here.